CoreScriptsRoot/LoadingScript.lua

```diff
@@ -762,3 +763,24 @@ game:GetService("ReplicatedFirst").RemoveDefaultLoadingGuiSignal:connect(handleR
 if game:GetService("ReplicatedFirst"):IsDefaultLoadingGuiRemoved() then
 	handleRemoveDefaultLoadingGui()
 end
+
+local UserInputServiceChangedConn;
```

Comment:
Why do we include a ";" at the end of it? I rarely see that after declaring the connection variable.